src/components/FullDetailView/AbstractFullDetailsView.js

```diff
@@ -32,7 +32,8 @@ export class AbstractFullDetailsView extends Component {
 
     return modalView ? (
       <Modal
-        closable={false}
+        closable={!!this.handleClose}
```

Comment:
This change meant to allow Modal to be closed if this.handleClose is defined.  `FullDetailPage`, the sole subclass of `AbstractFullDetailsView`, has handleClose.

Reply:
since `handleClose` is not defined in this class, not defined in `Component`, or being passed in via props, we can probably omit this and just keep it as false since it'll always be false